### PR TITLE
Fixed ctf Sound Events and Flag Icon

### PR DESCRIPTION
**Pull request by DBmaoha**
Norshtar before using a wrong sound name that causes enemy picking up/ capture flag to have no sounds, also there's no icon in minimap for flags
This will fix them

### CODE REVIEW

File: Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut [hunk 1]
```diff
@@ -446,6 +450,12 @@ void function OnPlayerEntersFlagReturnTrigger( entity trigger, entity player )
 		flag = file.imcFlag
 	else
 		flag = file.militiaFlag
+
+	if( !IsValid( flag ) )
+		return
+	
+	if( !IsValid( player ) )
+		return
```
Comment (uniboi) [inline Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut:458]:
```suggestion
	if( !IsValid( flag ) || !IsValid( player ) )
		return
```

File: Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut [hunk 2]
```diff
@@ -408,7 +412,7 @@ void function CaptureFlag( entity player, entity flag )
 	AddTeamScore( team, 1 )
 	AddPlayerScore( player, "FlagCapture", player )
 	player.AddToPlayerGameStat( PGS_ASSAULT_SCORE, 1 ) // add 1 to captures on scoreboard
-	SetRoundWinningKillReplayAttacker( player ) // set attacker for last cap replay
+	//SetRoundWinningKillReplayAttacker( player ) // set attacker for last cap replay
```
Comment (uniboi) [inline Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut:415]: Is there a reason for this?

**Review by uniboi — CHANGES_REQUESTED** [review]

File: Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut (hunk 2 above)
Comment (BobTheBob9) [reply Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut:415]: i'll merge if this is figured out @DBmaoha, from my perspective this looks like it'd break last cap replay

Comment (DBmaoha) [reply Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut:415]: someting I modified for my own use, forgot to reset it before pr

**Pull request merged**